exercises/concept/deep-thought/.docs/instructions.md

```diff
@@ -1,10 +1,14 @@
 # Errors
 
-In this exercise, we will be implementing the programming logic in a package `deepthought`,
-which checks whether an input equals to the absolute answer to life, the universe and everything.
+In this exercise, we will be implementing the programming logic in a package `deep_thought`,
```

Comment:
I'm not sure what the proper github etiquette is here? The package name should be one word to align with the Go standards, unless I'm missing another consideration. 

```suggestion
In this exercise, we will be implementing the programming logic in a package `deepthought`,
```